docs/config-reference.md

```
@@ -417,12 +417,12 @@ Benchmark configuration. The `type` field determines which benchmark runner is u
 | ----------------- | ---------------------------------------------- |
 | `manual`          | No benchmark (default), manual testing mode    |
 | `sa-bench`        | Throughput/latency serving benchmark           |
+e| `sglang-bench`    | SGLang bench_serving benchmark                 |
```

markdownlint-cli2 (0.20.0)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: trailing_only; Missing leading pipe

(MD055, table-pipe-style)